docs/src/main/asciidoc/kubernetes.adoc

```diff
@@ -674,7 +669,7 @@ The properties for configuring `docker` and `s2i` are also deprecated in favor o
 
 ==== Config group arrays
 
-Properties refering to config group arrays (e.g. kubernetes.labels[0], kubernetes.env-vars[0] etc) have been converted to maps, to align with the rest of the quarkus ecosystem.
+Properties referring to config group arrays (e.g. kubernetes.labels[0], kubernetes.env-vars[0] etc) have been converted to maps, to align with the rest of the quarkus ecosystem.
```

Comment:
Same here, can you check the case of `Quarkus` throughout the doc?

Reply:
Yup, fixes